test/models/job_test.exs

```diff
@@ -13,21 +13,21 @@ defmodule Community.JobTest do
   end
 
   test "validates the url" do
-    attributes = fields_for(:job, %{company_url: "something.com"})
+    attributes = params_for(:job, %{company_url: "something.com"})
```

Comment:
Maps works, but it's more typical to use a Keyword list. either way is fine, but I thought I'd mention it